nss/integration-tests/helper_test.go

```diff
@@ -65,7 +65,7 @@ func getentOutputForLib(t *testing.T, libPath, socketPath string, rustCovEnv []s
 	cmd.Env = append(cmd.Env,
 		"AUTHD_NSS_INFO=stderr",
 		// NSS needs both LD_PRELOAD and LD_LIBRARY_PATH to load the module library
-		fmt.Sprintf("LD_PRELOAD=%s:%s", libPath, os.Getenv("LD_PRELOAD")),
+		fmt.Sprintf("LD_PRELOAD=%s %s", libPath, os.Getenv("LD_PRELOAD")),
```

Comment:
Weird syntax considering that they use `:` as a separator on the other env right below... Are you sure the `:` doesn't work?

Reply:
Ah, historically it only supported spaces, but it seems that now it also handles colons, so let me re-use that, for consistency... (see also [`man ld.so`](https://manpages.ubuntu.com/manpages/noble/en/man8/ld.so.8.html))